test/cpp/jit/test_gpu.cpp

```diff
@@ -2661,6 +2671,25 @@ TEST(NVFuserTest, FusionUnaryOps_CUDA) {
         std::make_tuple(std::make_pair(ValType::TensorView, DataType::Float)));
   });
 
+  std::for_each(
+      ops_with_dtype.begin(), ops_with_dtype.end(), [](OpWithDTypeTuple& op) {
+        test_op(
+            /*blocks*/ 640,
+            /*threads*/ 64,
+            /*name*/ std::get<2>(op),
+            /*Aten Func   */
+            [&op](std::array<IValue, 1>& vals) {
+              return std::get<0>(op)(vals[0].toTensor(), c10::nullopt);
```

Comment:
For this test we just pass `c10::nullopt` as dtype. Should this test be changed?

Reply:
Let's try to make as few JIT changes as possible.

Reply:
Here is also the similar the problem that the test above expects function signatures of `Tensor (*)(const Tensor&)` for UnaryOps.

We can either disable it or update it. Have updated it for now and verified that it works.

Reply:
Got it. Updating it is great. Thanks for being so diligent.